apps/chatbot/src/bot/commands/version.ts

```diff
@@ -0,0 +1,18 @@
+import { readPackage } from "read-pkg";
+
+import type { CommandHandler } from "@/bot/commands/base";
+import { createBotCommand } from "@/bot/commands/base";
+
+const versionPromise = readPackage().then(({ version }) => version);
```

Comment:
Would the commit hash from git be more useful here? We can probably just shell out to `git rev-parse --short HEAD` to get it?

Reply:
Right now the chatbot runs in the docker without git installed. I am not sure if it makes sense to install git just for this or maybe we can write the ref to a file with GH actions instead?!

Reply:
Ah yeah -- I would pass it in as a build arg and then expose it as an env var for the bot?

Reply:
I got it running using a runtime env variable, but i think that should be fine as well?! 4157d95dfbcb6ae3b55598fe2d45dd2e127e1ad2

Reply:
Works for me 👍